src/platform/datapath_raw.c

```diff
@@ -270,6 +272,7 @@ CxPlatSocketDelete(
     _In_ CXPLAT_SOCKET* Socket
     )
 {
+    CxPlatDpRawOnSocketStateChange(Socket, FALSE);
```

Comment:
nit: why don't we call it CxPlatDpRawPlumbRulesOnSocket. CxPlatDpRawOnSocketStateChange seems a little weird to me. I can't easily tell what it does by just looking at the name.

Reply:
Ok. I can change the name.